testing/test_aux.out

```diff
@@ -10,11 +10,11 @@ RESULT info protocol.bacnet.version Protocol version: 1
 RESULT pass protocol.bacnet.pic The devices matches the PICS
 RESULT fail connection.mac_oui Manufacturer prefix not found!
 RESULT pass connection.mac_oui Manufacturer: Google found for address 3c:5a:b4:1e:8f:0a
-RESULT skip security.tls.v1 IOException unable to connect to server
+RESULT skip security.tls.v1 IOException unable to connect to server 
```

Comment:
I've run grep on the daq folder to find that the line of code which outputs this, "daq/subset/security/tlstest/src/main/java/Certs.java" (this may also address the other comment about the Host: xxxx line by looking at the last few commits on this Certs.java) was already outputting a space at the end even before this pull request. I don't think the code in this pull request modifies any of the tls tests.

It could just be that test_aux.out has not been updated in a while and incremental changes were added during travis testing for it to pass.

Hope this helps.